sci-physics/thepeg/thepeg-2.2.3-r1.ebuild

```diff
@@ -22,14 +22,14 @@ S="${WORKDIR}/${MY_P}"
 LICENSE="GPL-3"
 SLOT="0/30"
 KEYWORDS="~amd64 ~x86"
-IUSE="emacs fastjet hepmc java lhapdf static-libs test zlib"
+IUSE="emacs fastjet hepmc3 java lhapdf static-libs test zlib"
```

Comment:
In 2.3.0 this flag is enabled by default, lets do that here as well to also avoid surprises due to the renaming here.